scripts/rstlint.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Check for stylistic and formal issues in .rst and .py
+# files included in the documentation.
+#
+# 01/2009, Georg Brandl
```

Comment:
Use consistent copyright notice.

Reply:
If this is copy from another project, how should we do with the copyright notice? Add another on the top or reserve the old one?

Reply:
Simply rewrite as `Copyright (C) 2009 Georg Brandl` and mention external source in `README.md`.